cedar-dafny/difftest/main.dfy

```diff
@@ -394,7 +394,7 @@ module difftest.main {
           case "Record" =>
             var attrs :- getJsonField(body1, "attrs");
             var attrs1 :- deserializeField(attrs, "attrs", attrTypesFromProdJsonObjectRec);
-            Ok(Type.Record(attrs1))
+            Ok(Type.Record(RecordType(attrs1, false)))
```

Comment:
Is there no way to specify that a record is open/closed in the json representation?

Reply:
Not at this point. That will be part of partial schema validation.

Reply:
Ah right, makes sense.

Reply:
See my comment at the end about defaults being open/closed.